### ISSUE TITLE
Final release?

### ISSUE THREAD

**Issue by bluesky314** [rwightman/efficientdet-pytorch#19]
Are we getting closer to the final release of the weights that get equal accuracy to what the paper suggests? I think you have been working for over a month on this model now, could you also explain the difficulties you faced in reaching the same accuracy the paper claimed? Would be interested to learn from your experience on how to approach these models.

**Comment by rwightman** [comment]
Final? I couldn't say, still lots of potential improvements. At this point I feel it is working well. Results training from scratch have been closer to the official than any other impl out there (which usually can't get anywhere close) -- this was before fixing a weight init issue which improves convergence significantly. The model w/ bench + loss has been used by others in a Kaggle competition with an alternate (albumentations based) augmentation pipeline for good results. I'd say it's useable as is and by far the cleanest implementation.

I likely won't release any of the weights so far as they were done with the TF like same padding mode so there is no point. I will release weights once I get models  train on the non `tf_` versions without the same padding which I just added recently. I have a D1 training like that but it'll take a week. I have also created some experimental custom configs with other backbones ('MixDet') as this can use any model in my timm collection that has a feature extractor mode (so far any EfficientNet/MobileNetV3 family model which includes MixNet, MobileNetv2, MnasNet)

If you want to contribute money or GPU time for running more experiments let me know. It takes 4-5 days for me to train a D0, and 7-8 for a D1, I don't have the capability to train a D7, it'd likely cost $10-20k USD in cloud time if you are so inclined.
